decidim-core/lib/decidim/form_builder.rb

```diff
@@ -133,6 +133,39 @@ def check_box(attribute, options = {}, checked_value = "1", unchecked_value = "0
       end + error_and_help_text(attribute, options)
     end
 
+    # Public: Override so the date fields are rendered using foundation
+    # datepicker library
+    def date_field(attribute, options = {})
+      template = ""
+      template += @template.label(@object_name, attribute)
+      template += @template.text_field(@object_name, attribute, options.merge({
+        name: nil,
+        id: "date_field_#{@object_name}_#{attribute}",
+        data: { datepicker: '', startdate: object.send(attribute) }
+      }))
+      template += @template.hidden_field(@object_name, attribute)
+      template += error_and_help_text(attribute, options)
+      template.html_safe
+    end
+
+    # Public: Generates a timepicker field using foundation
+    # datepicker library
+    def datetime_field(attribute, options = {})
+      value = object.send(attribute)
```

Comment:
It's already tested on: https://github.com/AjuntamentdeBarcelona/decidim/pull/1207/files#diff-4e834bfefb3832d6fefd3aa5fac661ea